hack/vendor.sh

```diff
@@ -19,7 +19,15 @@ case $# in
 	;;
 # If user passed arguments to the script
 1)
-	eval "$(grep -E "^clone [^ ]+ $1" "$0")"
+	set +e
```

Comment:
Rather than using `set +e` (which disables completely the "failure protections" it provides), we should look into using either `set -o pipefail` (which I'm not sure will actually work here), or using a variable, which will force the `grep` to be evaluated outside the "complex expression", and thus let `set -e` fail it (or let us `if` it and provide a good message).

For example:
```bash
if ! cloneGrep="$(grep -E "^clone [^ ]+ $1" "$0")"; then
    echo >&2 "error: failed to find 'clone ... $1' in $0"
    exit 1
fi
eval "$cloneGrep"
```